packages/pn-pa-webapp/src/pages/ApiKeys.page.tsx

```diff
@@ -278,7 +278,11 @@ const ApiKeys = () => {
             {modal.view === ModalApiKeyView.ENABLE && (
               <ApiKeyModal
                 title={t('enable-api-key')}
-                subTitle={t('enable-warning', { apiKeyName: modal.apiKey?.name })}
+                subTitle={
+                  <Trans i18nKey="enable-warning" values={{ apiKeyName: modal.apiKey?.name }}>
+                    {t('enable-warning', { apiKeyName: modal.apiKey?.name })}
```

Comment:
The children in Trans component are used as default when translation fail (because key is not found).
This makes it difficult to understand which localizations are missing.
Do for all Trans used here.
Delete the default and try to use a wrong translation key, to confirm the above.